multiversx_sdk/controllers/smart_contract_controller.py

```diff
@@ -199,11 +173,11 @@ def create_transaction_for_execute(self,
 
         return transaction
 
-    def parse_execute(self, transaction_on_network: TransactionOnNetwork, function: Optional[str] = None):
-        pass
+    def parse_execute(self, transaction_on_network: TransactionOnNetwork, function: Optional[str] = None) -> List[Any]:
+        raise NotImplementedError("This method is not yet implemented")
```

Comment:
True. We'll see how to use this in the future:

https://github.com/multiversx/mx-chain-proxy-go/pull/446